source/postprocess/sea_level.cc

```diff
@@ -200,9 +202,7 @@ namespace aspect
       integral_ice_height = Utilities::MPI::sum (integral_ice_height, this->get_mpi_communicator());
       integral_topo_geoid = Utilities::MPI::sum (integral_topo_geoid, this->get_mpi_communicator());
 
-      const double sea_level_offset = -1./integral_ocean_mask*(1./density_water*integral_ice_height+integral_topo_geoid);
-
-      return sea_level_offset;
+      return -1./integral_ocean_mask*(1./density_water*integral_ice_height+integral_topo_geoid);
```

Comment:
This variable shadows a member variable of the class. This is not wrong, but it's confusing, so let's just avoid the use of the local variable altogether.